examples/chart/teleport-agent/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: "teleport"
```

Comment:
Are there healthz/readyz endpoints on Teleport to point liveness/readiness probes to?

Reply:
Added `/healthz` as liveness probe and `/readyz` as readiness probe
See https://goteleport.com/teleport/docs/admin-guide/#troubleshooting